### PR TITLE
rails activerecord crud study

### PR DESCRIPTION
**Pull request by apalmer0**

### CODE REVIEW

File: study.md [hunk 1]
```diff
@@ -95,11 +107,14 @@ would use to satisfy these requirements.
...
 ```
 
 Should you use seeds to create data to experiment with during development?
 
 ```md
-<!-- your response here -->
+You could, but seeds are really just meant for adding necessary initial data.
+Creating data for testing can be done with gems that are better suited to that
+task (Faker, for example). 
 ```
```
Comment (RealWeeks) [inline study.md:120]: Nice, how'd you know about faker?

**Comment by RealWeeks** [comment]
Nice Andrew.  Good job.  You clearly understand the reading, you're going to be setup for success when you make your own API.

File: study.md (hunk 1 above)
Comment (jrhorn424) [inline study.md:120]: Nice! I'm on the fence with `Faker`, since it's an API for creating objects... which is what Ruby does already... ;)

**Comment by jrhorn424** [comment]
Good stuff. :neckbeard: :octocat: :+1:

**Pull request closed without merging**